mw-webapp/src/model/firebaseCollection/time/Time.ts

```diff
@@ -0,0 +1,31 @@
+import {TimeUnit} from "src/model/businessModel/time/timeUnit/TimeUnit";
+
+/**
+ * What was done
```

Comment:
What)

Reply:
Change comment to "How long was the job done"